python/fastquant/strategies/base.py

```diff
@@ -226,14 +229,19 @@ def stop(self):
         last_date = str(self.datas[0].datetime.date(0))
         if self.channel:
             trigger_bot(
-                self.symbol, self.action, last_date,
+                self.symbol,
+                self.action,
+                last_date,
             )
 
     def start(self):
         # Used to signal setting the first iteration
         self.first_timepoint = True
 
     def next(self):
+
+        # add dividend to cash
+        self.broker.add_cash(self.datadiv)
```

Comment:
Nice, exactly where I'd put it :smile:

Reply:
@rafmacalaba Also I realized that this means dividends are automatically reinvested if the column exists. Perhaps we can add a `invest_div` bool argument that's set to `True` by default? Also open to keeping it as is if you think adding the div column is enough though.

Thoughts?